services/search/synonym_extractor.py

```diff
@@ -0,0 +1,63 @@
+import logging
+from requests_futures.sessions import FuturesSession  # pylint: disable=import-error
+import constants
+
+
+class SynonymExtractor:
+    """
+    Wrapper class for requests_futures methods
+    """
+    def __init__(self, max_synonyms):
+        self.max_synonyms = max_synonyms
+        self.session = FuturesSession()
+
+    def get_responses_as_json(self, word_arr):
+        """
+        create an asynchronous request list using requests-futures
+        Attributes:
+            word_arr: A list of words which each is a different request
+            session: A requests-futures session
+        Return:
+            A list of requests for every word
+        """
+        responses = []
+        for word in word_arr:
+            responses.append(self.session.get(constants.SPANISH_API_URL+word))  # pylint: disable=no-member
+
+        responses_json = [self.get_response_json(resp) for resp in responses]
+        return responses_json
+
+    def get_response_json(self, resp):
```

Comment:
[nit] I believe you can make this private to the class now by renaming it `__get_response_json`: https://www.tutorialspoint.com/What-does-double-underscore-prefix-do-in-Python-variables#:~:text=If%20your%20class%20is%20intended,mangled%20into%20the%20attribute%20name.